packages/TorneloScoresheet/src/hooks/appMode/arbiterResultDisplayState.ts

```diff
@@ -0,0 +1,36 @@
+import { useContext } from 'react';
+import {
+  AppMode,
+  AppModeState,
+  ArbiterResultDisplayMode,
+} from '../../types/AppModeState';
+
+type arbiterResultDisplayStateHookType = [
+  ArbiterResultDisplayMode,
+  {
+    goToResultDisplayMode: () => void;
+  },
+];
+
+export const makeUseArbiterResultDisplayState =
+  (
+    context: React.Context<
+      [AppModeState, React.Dispatch<React.SetStateAction<AppModeState>>]
+    >,
+  ): (() => arbiterResultDisplayStateHookType | null) =>
+  (): arbiterResultDisplayStateHookType | null => {
+    const [appModeState, setAppModeState] = useContext(context);
+    if (appModeState.mode !== AppMode.ArbiterResultDisplay) {
+      return null;
+    }
+
+    const goToResultDisplayMode = (): void => {
+      setAppModeState({
+        mode: AppMode.ResultDisplay,
+        pairing: appModeState.pairing,
+        result: appModeState.result,
```

Comment:
same deal here, we can just set it to `{...appModeState, mode: AppMode.ResultDisplay}`

Reply:
Though in this case and arbiterTablePairingState we are not changing anything, but writing it this way makes it so that if we change the state later on, we wont have to change this code here since the whole state will be copied over